src/main/webapp/jsp/instructorFeedbackAjaxRemindParticularStudentsModal.jsp

```diff
@@ -26,7 +26,7 @@
       <span class="icon-sort unsorted"></span>
     </th>
   </tr>
-  <c:forEach items="${data.responseStatus.noResponse}" var="userToRemindEmail">
+  <c:forEach items="${data.responseStatus.getStudentsWhoDidNotRespondSorted()}" var="userToRemindEmail">
```

Comment:
You'll find that changing this to `${data.responseStatus.studentsWhoDidNotRespondSorted}` will work out of the box (it automatically calls `getStudentsWhoDidNotRespondSorted()`).